Corcav.Behaviors/Library/EventToCommand.cs

```diff
@@ -148,7 +148,7 @@ private void AddEventHandler(EventInfo eventInfo, object item, Action<EventArgs>
         /// <param name="e">The EventArgs value accompanying the Event</param>
         private void OnFired(EventArgs e)
 		{
-            object returnParam = this.CommandParameter ?? e;
+            object returnParam = e ?? this.CommandParameter;
```

Comment:
I'll take a look at this and see if I can reproduce and make a fix for it.  Might be better to add a property to determine whether we want EventArgs to be sent.

Thoughts?